plugins/view-transitions/includes/settings.php

```diff
@@ -191,17 +203,22 @@ function plvt_register_setting(): void {
  * @global array<string, mixed> $_wp_theme_features Theme support features added and their arguments.
  */
 function plvt_apply_settings_to_theme_support(): void {
-	global $_wp_theme_features;
+	global $plvt_has_theme_support_with_args, $_wp_theme_features;
```

Comment:
Missing global doc for `$plvt_has_theme_support_with_args`

Reply:
Fixed in https://github.com/WordPress/performance/pull/2037/commits/c10373379c69e8ac0a42f9b80428a17d18aa40b5